src/legacy/core_plugins/kibana/public/discover/plugin.ts

```diff
@@ -102,6 +137,9 @@ export class DiscoverPlugin implements Plugin<DiscoverSetup, DiscoverStart> {
     plugins.kibanaLegacy.registerLegacyApp({
       id: 'discover',
       title: 'Discover',
+      // only register the updater in once app, otherwise all updates would happen twice
```

Comment:
nit: This comment is not applicable here (it's just important for dashboard because technically there are two apps there for listing and detail page)